src/KeyVault/KeyVault/Commands/Certificate/ImportAzureKeyVaultCertificate.cs

```diff
@@ -169,6 +199,11 @@ public override void ExecuteCmdlet()
             }
         }
 
+        private bool IsPemFile(string filePath)
+        {
+            return ".pem".Equals(Path.GetExtension(FilePath), StringComparison.OrdinalIgnoreCase);
```

Comment:
filePath is the method parameter, FilePath is a property.
```suggestion
            return ".pem".Equals(Path.GetExtension(filePath), StringComparison.OrdinalIgnoreCase);
```